content/docs/tinacms-context.md

```diff
@@ -129,9 +129,72 @@ export const getStaticProps = async () => {
 - `query` and `variables`: These are the same values that you would use for the [backend data-fetching](/docs/features/data-fetching/).
 - `data`: This is the production value that gets passed through to the response unchanged in production.
 
+## The `RouteMapping` Plugin
+
+The `RouteMapping` Plugin is used in the Collection List pages to route to a Document's "Contextual Editing" view rather than the default "Fullscreen Editing" view.
```

Comment:
Just finding the wording of "route to" a bit confusing, could we use the word "link" or "navigate".

Reply:
I find "Collection List pages" also a bit confusing. Is that referring to the admin?